src/pybind/rados/rados.pyx

```diff
@@ -2548,6 +2549,45 @@ cdef class Ioctx(object):
         if ret < 0:
             raise make_ex(ret, "error flushing")
 
+    @requires(('object_name', str_type), ('cmp_buf', bytes), ('offset', int))
+    def aio_cmpext(self, object_name, cmp_buf, offset=0, oncomplete=None):
+        """
+        Asynchronously compare an on-disk object range with a buffer
+        :param object_name: the name of the object
+        :type object_name: str
+        :param cmp_buf: buffer containing bytes to be compared with object contents
+        :type cmp_buf: bytes
+        :param offset: object byte offset at which to start the comparison
+        :type offset: int
+        :param oncomplete: what to do when the write is safe and complete in memory
+            on all replicas
+        :type oncomplete: completion
+
+        :raises: :class:`TypeError`
+        returns: 0 - on success, negative error code on failure,
+                 (-MAX_ERRNO - dismatch_off) on dismatch
```

Comment:
please update all instances of s/dismatch/mismatch

Reply:
update, please check, thanks.